src/themes/cerulean/collections/menu.overrides

```diff
@@ -1,7 +1,79 @@
-.ui.secondary.menu .item {
-  color: @secondaryItemColor;
+#gradient {
+    .vertical-three-colors(@start-color: #00b3ee; @mid-color: #7a43b6; @color-stop: 50%; @end-color: #c3325f) {
+        background-image: -webkit-linear-gradient(@start-color, @mid-color @color-stop, @end-color);
+        background-image: -o-linear-gradient(@start-color, @mid-color @color-stop, @end-color);
+        background-image: linear-gradient(@start-color, @mid-color @color-stop, @end-color);
+        background-repeat: no-repeat;
+        filter: e(%("progid:DXImageTransform.Microsoft.gradient(startColorstr='%d', endColorstr='%d', GradientType=0)",argb(@start-color),argb(@end-color))); // IE9 and down, gets no color-stop at all for proper fallback
+    }
 }
 
-.ui.tabular.menu .item {
-  color: @secondaryItemColor;
+.btn-shadow(@color) {
```

Comment:
So I guess it's a mixin in LESS/SASS language.

It's OK to use this. One thing I prefer, please use full word instead of abbreviations, just to make it more like semantic-ui's `ui button` instead of bootstrap's `btn-`. For `btn-shadow`, prefer `button-shadow`.